File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/sector/Sector.java
package com.api.hospitalsystem.model.sector;

public enum Sector {

    Room("Quarto"),
    Infirmary("Enfermaria"),
    BirthRoom("Sala de parto"),
    Nursery("Berçário"),
    ApartmentPostpartum("Apartamento para o pós-parto"),
    Emergency("Pronto-socorro"),
    Icu("UTI"),
    Surgey("Cirurgia"),
    Adm("Administração");

    private final String description;

    Sector(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}

File: app/hospital-system/src/app/module/models/src/sector/sector.ts
import { describe } from '@module/utils/functions/enum';

export enum Sector {
  Room = 0,
  Infirmary = 1,
  BirthRoom = 2,
  Nursery = 3,
  ApartmentPostpartum = 4,
  Emergency = 5,
  Icu = 6,
  Surgey = 7,
  Adm = 8,
}

describe(Sector, {
  Room: 'Quarto',
  Infirmary: 'Enfermaria',
  BirthRoom: 'Sala de parto',
  Nursery: 'Berçário',
  ApartmentPostpartum: 'Apartamento para o pós-parto',
  Emergency: 'Pronto-socorro',
  Icu: 'UTI',
  Surgey: 'Cirurgia',
  Adm: 'Administração',
});
